.github/workflows/main.yaml

```diff
@@ -30,3 +30,25 @@ jobs:
       run: pip install tox
     - name: Lint
       run: tox -vve lint
+
+  integration-test:
+    name: Integration test with VMWare
+    runs-on: self-hosted
+    timeout-minutes: 360
```

Comment:
let's expect to finish in 90m.  Especially if you eliminate the Flaky and Slow tests